changelog.d/20230620_150848_kurtmckee_add_run_delete_sc_18418.md

```diff
@@ -0,0 +1,3 @@
+### Enhancements
+
+* Support deleting runs in the Flows service.
```

Comment:
Let's enhance this changelog to list the new command's name, `globus flows runs delete`.

Or `globus flows run delete` if we take that naming approach.

Reply:
Done.